config/crd/bases/capsule.clastix.io_tenants.yaml

```diff
@@ -601,7 +605,20 @@ spec:
                 type: string
               type: object
             owner:
-              type: string
+              description: OwnerSpec defines tenant owner name and kind
+              properties:
+                kind:
+                  enum:
+                  - User
+                  - Group
+                  - ServiceAccount
```

Comment:
Since `ServiceAccount` are namespace scoped resources, I guess this can be dropped.